src/cli.ts

```diff
@@ -0,0 +1,142 @@
+      return false;
+    }
+    return true;
+  }
+  /**
+   *
+   * @param path the actual path for the README.md.tsx file to be output
+   */
+  public CreateReadmeJsx(path: string): void {
+    console.log('\n')
+    console.log('Creating README.md.tsx file in with the following path ' + path);
+    //make sure the path is well written
+    if (path.charAt(path.length - 1) !== '/') {
+      path = path + '/';
+      console.log(path)
+    }
+    // we check if the path given exists
+    if (!fs.existsSync(path)) {
+      console.error('Path does not exist')
+      throw new Error('Path specified does not exist');
+    } else {
+      const template_readme_file = `// We need to tell the JSX transpiler that in this file,
+// instead of React we use the custom createElement and Fragment
+// functions from jsx-readme
+/* @jsx MD */
+/* @jsxFrag Fragment */
+import type { Component } from "..";
+import MD, {
+  BadgesFromPkg,
+  CodecovBadge,
+  DescriptionFromPkg,
+  ExamplesFromPkg,
+  Fragment,
+  GithubWorkflowBadge,
+  HomepageFromPkg,
+  renderToFile,
+  TitleFromPkg,
+  DiscordBadge,
+  HacktoberfestBadge,
+  } from "..";
+import { Heading, InlineCode, LineBreak } from "jsx-md";
+import pkg from "./package.json";
+
+const Readme: Component = () => (
+  <Fragment>
+    {/* Create a header with title, badges and description inferred from package.json */}
+    <TitleFromPkg pkg={pkg} />
+    <BadgesFromPkg pkg={pkg} />
+    {/* Add additional badges. */}
+    <CodecovBadge pkg={pkg} />
+    <GithubWorkflowBadge pkg={pkg} workflowName="Build and deploy" />
+    <DiscordBadge
+      inviteLink="https://discord.com/invite/X9HRSK5"
+      serverId="750063320614174871"
+    />
+    <HacktoberfestBadge
+      pkg={pkg}
+      year={2020}
+      suggestionLabel="good first issue"
+    />
+    <LineBreak />
+    <DescriptionFromPkg pkg={pkg} />
+    {/* You can use the components from jsx-md to build custom markdown. */}
+    <Heading level={2}>Installation</Heading>
+    Add <InlineCode>jsx-readme</InlineCode> to your{" "}
+    <InlineCode>devDependencies</InlineCode> and install it. I recommend using
+    it with <InlineCode>ts-node</InlineCode>. Then all you need to do is run {" "} <InlineCode>npx jsx-readme init [destinationPath]</InlineCode>
+    where the {" "} <InlineCode>[destinationPath]</InlineCode> is where the {" "} <InlineCode>README.md.tsx</InlineCode> file will be outputed and once done
+    you run the {" "} <InlineCode>README.md.tsx</InlineCode> via{" "}
+    <InlineCode>ts-node</InlineCode> whenever you want to create a new version
+    of the <InlineCode>README</InlineCode>.
+    <LineBreak />
+    <LineBreak />
+    {/* Create an example section based on all files from the example directory set up in package.json */}
+    <ExamplesFromPkg pkg={pkg} />
+    {/* Create a section linking to the homepage from package.json */}
+    <HomepageFromPkg pkg={pkg} />
+  </Fragment>
+);
+
+void renderToFile("./README.md", <Readme />);
```

Comment:
Please simplify this so that it is directly usable without any changes, e. g. by removing things only relevant to jsx-readme or badges that might not be relevant.